### PR TITLE
fix(apidom-ns-openapi-3-0): semantic highlighting for responses

### PR DESCRIPTION
**Pull request by robert-hebel-sb**
Fix highlighting default responses by removing default key from fixed parameters

### How Has This Been Tested?
unit test + tested manually on local env

### Screenshots (if appropriate):
Screenshot 2025-07-04 at 16 30 06

### My PR contains...

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [ ] are not breaking changes.

### Documentation
- [ ] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/apidom-ns-openapi-3-0/src/refractor/specification.ts [hunk 1]
```diff
@@ -299,9 +298,7 @@ const specification = {
         },
         Responses: {
           $visitor: ResponsesVisitor,
-          fixedFields: {
-            default: ResponsesDefaultVisitor,
```
Comment (robert-hebel-sb) [inline packages/apidom-ns-openapi-3-0/src/refractor/specification.ts:303]: this, shouldn't be changed hence changing status to darft
